src/Provider/Provider.Web/Views/CloseVacancy/Close.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@model Esfa.Recruit.Provider.Web.ViewModels.CloseViewModel
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name=@PartialNames.ValidationSummary model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState }' />
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">
+            @Model.Title
+            <span class="govuk-caption-xl">VAC<span>@Model.VacancyReference</span>
+        </h1>
+
+        <h2 class="govuk-heading-m govuk-!-margin-top-0">Closing a vacancy</h2>
+        <p class="govuk-body">By continuing with this action you will be closing this vacancy to further applications on Find an apprenticeship.</p>
+        <p class="govuk-body">It may take up to an hour to take effect on Find an apprenticeship.</p>
+        <div class="govuk-inset-text govuk-body">This action cannot be undone.</div>
+        <form asp-route="@RouteNames.CloseVacancy_Post">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--m govuk-!-margin-top-6 govuk-!-margin-bottom-6">
+                <h3 class="govuk-fieldset__heading">
+                    Do you want to create a new vacancy or clone an existing vacancy?
```

Comment:
We cannot ask them to clone a vacancy on the close vacancy page!

Reply:
Good spot, Sachin!